config.json

```diff
@@ -1,264 +1,1802 @@
 {
   "cards": [
     {
-      "resource_cost": 0, 
+      "cities": 6, 
       "cost": 44, 
-      "resource": "green", 
-      "cities": 6
+      "resource_cost": 0, 
+      "resources": [
+        "green"
+      ], 
+      "type": "light"
```

Comment:
Decided to just programatically add in the type of each card, because these are never going to change, so it's easier to just put it in once.